### ISSUE TITLE
This Deployment has been disabled.  Your connection is working correctly.  Vercel is working correctly. 

### ISSUE THREAD

**Issue by trevorkhaimia** [ksalpern/ai-image-generator#1]
Maybe you deleted the project , or changed the url it isnt working

**Comment by ksalpern** [comment]
> Maybe you deleted the project , or changed the url it isnt working

Yeah, the link doesn't work because I didn't pay for hosting, but you can try to start the project locally of [see](https://www.youtube.com/watch?v=a6kESfs-Mzs) how it used to work
